@xen-orchestra/web/src/components/account-menu/AccountMenu.vue

```diff
@@ -8,38 +8,83 @@
         @click="open($event)"
       />
     </template>
-    <MenuItem icon="fa:book">
+    <MenuItem class="icon" icon="fa:circle-user">
+      <a class="link typo-body-bold-small" href="/#/user" rel="noopener noreferrer" target="_blank">
+        {{ t('account') }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
+      </a>
+    </MenuItem>
+
+    <MenuItem class="icon" icon="fa:satellite">
+      <a class="link typo-body-bold-small" href="/#/xoa/licenses" rel="noopener noreferrer" target="_blank">
+        {{ t('license-name', { name: 'XOA' }) }}
```

Comment:
change color of licence:

```suggestion
          <span>{{ t('license-name') }}</span>
          <span>XOA</span>
```

Reply:
```suggestion
     {{ t('license') }}
    <span class="license-name">{{ 'XOA' }}</span>
```
Something like that seems more appropriate :thinking:

Reply:
After discussion with @pdonias , no need to touch this part